modules/nf-core/longstitch/main.nf

```diff
@@ -0,0 +1,55 @@
+process LONGSTITCH {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::longstitch=1.0.5"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/longstitch:1.0.5--hdfd78af_0':
+        'biocontainers/longstitch:1.0.5--hdfd78af_0' }"
+
+    input:
+    tuple val(meta) , path (assembly, stageAs: "assembly.fa")
+    tuple val(meta2), path (reads   , stageAs: "reads.fq.gz")
+    val mode
+    val genome_size
```

Comment:
Optional args should come in through `ext.args`. Only mandatory values ( and mandatory + optional files ) should be supplied through `input:`